sig-scheduling/annual-report-2022.md

```diff
@@ -4,15 +4,15 @@
 
 1. What work did the SIG do this year that should be highlighted?
 
-   -
-   -
-   -
+   - Improvements to PodTopologySpread API: adding more knobs to control spreading behavior (introducing minDomains and matchLabelKey).
+   - Adding more hooks to simplify integrations with external schedulers (mutable pod scheduling directives, pod scheduling readiness)
+
 
 2. What initiatives are you working on that aren't being tracked in KEPs?
 
-   -
-   -
-   -
+   - Performance improvements
+   - Code refactorings and cleanups
+
```

Comment:
I would highlight Kueue, in addition to it being in the subprojects section.

Reply:
I think we also can add the part of the Integration of Kubeflow and scheduler-plugins.

Reply:
done.